examples/with-next-app-i18n/app/[locale]/layout.tsx

```diff
@@ -1,15 +1,21 @@
 import { NextIntlClientProvider } from 'next-intl';
 import type { Locale } from '@rainbow-me/rainbowkit';
 import { Providers } from './providers';
- 
+import { unstable_setRequestLocale } from 'next-intl/server';
+
 export function generateStaticParams() {
   return [{ locale: 'en-US' }, { locale: 'zh-CN' }];
 }
 
-export default async function LocaleLayout(
-  { children, params: { locale }}: 
-  { children: React.ReactNode, params: { locale: Locale} }
-) {
+export default function LocaleLayout({
+  children,
+  params: { locale },
+}: {
+  children: React.ReactNode;
+  params: { locale: Locale };
+}) {
+  unstable_setRequestLocale(locale);
```

Comment:
Temporary. More info [here](https://next-intl-docs.vercel.app/docs/getting-started/app-router#add-unstable_setrequestlocale-to-all-layouts-and-pages)